test/models/local_restriction_test.rb

```diff
@@ -0,0 +1,55 @@
+describe LocalRestriction do
+  let(:restriction) { described_class.new("E08000001") }
+
+  before do
+    LocalRestriction.any_instance.stubs(:file_name).returns("test/fixtures/local-restrictions.yaml")
+  end
+
+  it "returns the area name" do
+    assert_equal "Tatooine", restriction.area_name
+  end
+
+  it "returns the alert level" do
+    assert_equal 4, restriction.alert_level
+  end
+
+  it "returns the guidance" do
+    guidance = restriction.guidance
+    assert_equal "These are not the restrictions you are looking for", guidance["label"]
+    assert_equal "guidance/tatooine-local-restrictions", guidance["link"]
+  end
+
+  it "returns the extra restrictions" do
+    assert_nil restriction.extra_restrictions
+  end
+
+  it "returns nil values if the gss code doesn't exist" do
+    restriction = described_class.new("fake code")
+    name = restriction.area_name
+    guidance = restriction.guidance
+    assert_nil name
+    assert_nil guidance
```

Comment:
```suggestion
    assert_nil restriction.area_name
    assert_nil restriction.guidance
```

Reply:
Could have sworn I'd done this. Put this in now